keel-core/src/main/kotlin/com/netflix/spinnaker/keel/pause/Pause.kt

```diff
@@ -0,0 +1,18 @@
+package com.netflix.spinnaker.keel.pause
+
+import java.time.Instant
+
+/**
+ * A pause in an application or resource's actuation, requested by a user.
+ */
+data class Pause(
+  val scope: PauseScope,
+  val name: String,
```

Comment:
what does `name` means?

Reply:
The "name" of the thing that was paused (resource or application). It's the name of the column in the database as well. It's more like an identifier, really, but I haven't changed that table, just added the model.